plugins/uptimerobot/src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ContentHeader,
+  Content,
+  Header,
+  HeaderLabel,
+  Progress,
+  Page,
+  SupportButton,
+  useApi,
+} from '@backstage/core';
+import { Alert } from '@material-ui/lab';
+import { BackstageTheme } from '@backstage/theme';
+import { Button, Grid, makeStyles } from '@material-ui/core';
+import { MonitorTable } from '../MonitorTable';
+import { uptimerobotApiRef } from '../../api';
+import { useAutoUpdatingRequest } from '../../hooks';
+import LaunchIcon from '@material-ui/icons/Launch';
+import React from 'react';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  button: {
+    marginTop: `${theme.spacing(2)}px`,
+  },
+}));
+
+export const HomePage = () => {
+  const classes = useStyles();
+
+  const uptimerobotApi = useApi(uptimerobotApiRef);
+  const { value, loading, error } = useAutoUpdatingRequest(() =>
+    uptimerobotApi.getAllMonitors(),
+  );
+
+  return (
+    <Page themeId="tool">
+      <Header
+        title="UptimeRobot"
+        subtitle="UptimeRobots monitors if your site is available."
+      >
+        <HeaderLabel label="Owner" value="@simonknittel" />
+        <HeaderLabel label="Lifecycle" value="Alpha" />
+      </Header>
+
+      <Content>
+        <ContentHeader
+          title="Overview"
+          description={`This table lists all monitors accessible by your API keys. It automatically refreshes every ${uptimerobotApi.getUpdateInterval()} seconds.`}
+        >
+          <SupportButton>
+            UptimeRobots monitors if your site is available.
+          </SupportButton>
+        </ContentHeader>
+
+        {loading && <Progress />}
+        {error && <Alert severity="error">{error.message}</Alert>}
+
+        {value && (
+          <Grid container spacing={3} direction="column">
+            <Grid item>
+              <MonitorTable monitors={value.monitors} />
+            </Grid>
+          </Grid>
+        )}
+
+        <Button
```

Comment:
should use the Button in `@backstage/core` ideally, so we can standardise styling and behaviors

Reply:
This button links to an external page. As far as I can see the `<Button />` component from `@backstage/core` doesn't support the `href` property. Other example which I can find throughout the code base all use the button from `@material-ui/core` in that case. Am I missing something here?

Reply:
Hm it should support the same props as the MUI button and also react router Link!

https://github.com/backstage/backstage/blob/master/packages/core/src/components/Button/Button.tsx#L22

Anyway, let's skip this. It seems to be early days for the core button and we can always adjust later

Reply:
With that button I'm getting the following TypeScript error:
```
Property 'to' is missing in type '{ children: string; href: string; endIcon: Element; rel: string; color: "primary"; variant: "contained"; className: string; }' but required in type 'Pick<Props, "id" | "name" | "key" | "color" | "translate" | "hidden" | "size" | "style" | "disabled" | "value" | "children" | "form" | "slot" | "title" | "type" | "className" | ... 278 more ... | "to">'.
```